caravel/templates/caravel/basic.html

```diff
@@ -21,7 +21,7 @@
 
   <body>
     {% block navbar %}
-      {% if not viz or viz.request.args.get("standalone") != "true" %}
+      {% if not dash_standalone and (not viz or viz.request.args.get("standalone") != "true") %}
```

Comment:
you could also have a more generic "show_navbar" argument and pass it explicitly in both views.